tv-shows/src/app/components/core/Button/Button.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+
+import { Button } from '@chakra-ui/react';
+import { IReview } from '../../../typings/Review.type';
+
+interface IStyledButtonProps {
+	label: string;
+	type?: 'button' | 'submit';
+	review?: IReview;
+	onRemoveReview?: (review: IReview) => void;
```

Comment:
I don't thing that `onRemoveReview` is a good thing to have in the core button interface. This would be onClick, and the way you could do that is:

`interface IStyledButtonProps extends ButtonProps {
label: string }

where ButtonProps is imported from chakra.

Looking further, you don't have any specific props that are different form Chakras button component. What you could do is:
```
export const StyledButton = ({children, ...rest}: ButtonProps) => {
return  <Button
//your styling
{...rest}
> {children}</Button>
}
```

This way you pass the children and don't need the label prop.
Also, you spread all of the props to the Chakra Button component.